### PR TITLE
Update project_organization.rst:(importing section)fix typo, improve grammar

### PR DESCRIPTION
**Pull request by andy5995**
various grammar, spelling corrections

### CODE REVIEW

File: getting_started/workflow/project_setup/project_organization.rst [hunk 1]
```diff
@@ -51,12 +51,12 @@ to store built levels that use them.
...
 
-Because of this, assets are now imported from within the project
-folder transparently.
+Because of this, assets are now transparently imported from within the project
+folder.
 
-If a folder shouldn't be imported into Godot an exception can be made with a
-.gdignore file.
+If a folder shouldn't be imported into Godot, an exception can be made with a
+.gitignore file.
```
Comment (LikeLakers2) [inline getting_started/workflow/project_setup/project_organization.rst]: Actually, `.gdignore` is the correct filename. This sentence reference's *Godot's import process*, not something with Git.

**Comment by LikeLakers2** [comment]
Also, for the sake of the changelog, your commit name should be more specific than "Update project_organization.rst".

**Comment by andy5995** [comment]
> Also, for the sake of the changelog, your commit name should be more specific than "Update project_organization.rst".

Ok. What specifically would you suggest in the future for a change like that?

**Comment by LikeLakers2** [comment]
>Ok. What specifically would you suggest in the future for a change like that?

Typically, just explain what you're doing. Even something like your PR title will do: "Fix typo and improve grammar within project_organization.rst"

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**